client/app/telecom/telephony/line/calls/simultaneousLines/trunk/telecom-telephony-line-trunk-calls-simultaneousLines.html

```diff
@@ -0,0 +1,239 @@
+<section class="telephony-group-line-simultaneousLines">
+
+    <header>
+        <section-back-link section-back-link-to-state="telecom.telephony.line.calls" section-back-link-title="{{ 'telephony_line_calls_back_link' | translate }}">
+        </section-back-link>
+        <h1 data-translate="telephony_line_actions_line_calls_simultaneous_line_title"></h1>
+    </header>
+
+    <div class="text-center" data-ng-if="TrunkRatesSimultaneousLinesCtrl.trunkRates.loading">
+        <oui-spinner></oui-spinner>
+    </div>
+
+    <div class="telecom-section-content" data-ng-hide="TrunkRatesSimultaneousLinesCtrl.trunkRates.loading">
+
+        <toast-message></toast-message>
+
+        <div class="alert alert-info"
+             role="alert">
+             <p data-translate="telephony_line_actions_line_calls_trunk_simultaneous_info1"></p>
+             <p data-translate="telephony_line_actions_line_calls_trunk_simultaneous_info2"></p>
+             <p data-translate="telephony_line_actions_line_calls_trunk_simultaneous_info3"></p>
+        </div>
+
+        <p data-ng-if="TrunkRatesSimultaneousLinesCtrl.trunkRates.current.quantity > 100" data-translate="telephony_line_actions_line_calls_simultaneous_line_support"></p>
+
+        <div data-ng-if="TrunkRatesSimultaneousLinesCtrl.trunkRates.current.quantity <= 100 && !TrunkRatesSimultaneousLinesCtrl.trunkRates.loading">
+            <form name="simultaneousLinesCtrlForm"
+                  data-ng-if="!TrunkRatesSimultaneousLinesCtrl.trunkRates.order"
+                  data-ng-submit="TrunkRatesSimultaneousLinesCtrl.orderPacks()">
+
+                <div class="text-center" data-ng-if="!TrunkRatesSimultaneousLinesCtrl.contracts && TrunkRatesSimultaneousLinesCtrl.loading.order">
+                    <oui-spinner></oui-spinner>
+                </div>
+
+                <div class="row my-4">
+                    <div class="col-xs-12 col-md-6">
```

Comment:
`col-xs-12` is useless here.

Your are not aligned with the Bootstrap grid system here, please refer to https://github.com/twbs/bootlint/wiki/E014 for instance :)